example/android/app/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,7 @@
     android:icon="@mipmap/ic_launcher">
     <activity android:name=".MainActivity"
       android:launchMode="singleTop"
+        android:exported="true"
```

Comment:
What does this do?

Reply:
This is required from Android SDK 31 and upwards, and tells Android that .MainActivity can be executed from outside the application.